src/main/java/service/ShopService.java

```diff
@@ -0,0 +1,8 @@
+package service;
```

Comment:
```suggestion
package core.basesyntax.service;
```
all your sub packages should be placed in basesyntax package (fix it everywhere)